subsys/net/ip/net_if.c

```diff
@@ -4932,14 +4932,71 @@ void net_if_call_link_cb(struct net_if *iface, struct net_linkaddr *lladdr,
 	k_mutex_unlock(&lock);
 }
 
-static bool need_calc_checksum(struct net_if *iface, enum ethernet_hw_caps caps)
+static bool need_calc_checksum(struct net_if *iface, enum ethernet_hw_caps caps,
+	enum net_if_checksum_type chksum_type)
```

Comment:
nit: Please align with the opening bracket.

Reply:
Done